hyppo/independence/hhg.py

```diff
@@ -194,10 +264,33 @@ def test(self, x, y, reps=1000, workers=1, random_state=None):
         check_input = _CheckInputs(x, y, reps=reps)
         x, y = check_input()
 
-        x, y = compute_dist(x, y, metric=self.compute_distance, **self.kwargs)
-        self.is_distance = True
+        # Fast HHG Test
+        if self.auto:
+            pointer = (np.mean(x, axis=0), np.mean(y, axis=0))
+            zx, zy = pointer
+            zx = np.array(zx).reshape(1, -1)
+            zy = np.array(zy).reshape(1, -1)
+            xin = np.concatenate((zx, x))
+            yin = np.concatenate((zy, y))
+            distx, disty = compute_dist(
+                xin, yin, metric=self.compute_distance, **self.kwargs
+            )
+            # take first row of distance matrix (distance from center point)
```

Comment:
not sure how the first row achieves this

Reply:
I believe we talked about this.